### PR TITLE
Sync from main subgraph repository

### PR DESCRIPTION
**Pull request by refugene**
This PR updates the metrics subgraph from the main subgraph repo, which it was forked from.

### CODE REVIEW

**Review by nup9151f — APPROVED** [review]

File: networks/local.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "network": "mainnet",
-  "accessKey": {
-    "address": "0x44b6317365fb117c540dccfd29348f94d5bbffdc",
-    "startBlock": 0
+  "AppFactory": {
+    "startBlock": 3,
```
Comment (tinypell3ts) [inline networks/local.json:4]: Why is the `startBlock` 3 here? Can it go back to `0`?

Comment (refugene) [reply networks/local.json:4]: I took the start block from contracts deployment run-latest.json
Can set it back to 0 yes

**Comment by george-openformat** [comment]
Yikes, apologies @refugene for not reviewing this sooner. I'm must admit I'm slightly confused as to what is going on here. Is the metrics subgraph the exactly the same as the main subgraph just with added metric parameters? Or is there more going on here?

It feels like if this is going to be in a separate repo it should have the sole responsibility to index "metrics" and everything else should be left to the main subgraph. Therefore this should also be a seperate subgraph deployment.

Would be really helpful to have a meeting and get up to speed with thought process. 

Then if necessary we can go through merge conflicts together because that is going to be painful. :sob:

**Review by george-openformat — CHANGES_REQUESTED** [review]
is keeping up with main necessary? 

please see other comment

**Comment by refugene** [comment]
> is keeping up with main necessary?
> 
> please see other comment

Hi @george-openformat I have closed this PR. As you say this is not necessary and we also discussed it with Andy

**Pull request closed without merging**